Esterni/AnalisiDeiRequisiti/components/descrizioneprodotto.tex

```diff
@@ -0,0 +1,10 @@
+\section{Descrizione del prodotto}
+\subsection{Caratteristiche del prodotto}
+Lo scopo del capitolato C7 è la creazione di un algoritmo solido ed efficente in grado di garantire il salvataggio in cloud e la sincronizzazione dei cambiamenti presenti in cloud. Inoltre è richiesta anche la creazione di una interfaccia grafica multipiattaforma. Il progetto deve dunque funzionare sui principali sistemi operativi desktop quali Windows, MacOS e Linux senza richiedere l'installazione manuale di ulteriori prodotti per il corretto funzionamento. 
+\subsection{Obiettivo del prodotto}
+L'obbiettivo del progetto è la realizzazione di un modulo per la piattaforma \glo{Zimbra}, che abbia le caratteristiche sopra descritte, e che si appoggi per il suo funzionamento al modulo \glo{Zextras Drive}.
+Nello specifico lo scopo finale del prodotto è quello di fornire agli utenti un servizio che automatizzi il salvataggio in cloud e la sincronizzazione con il cloud stesso.
```

Comment:
Forse cambiare
"... che automatizzi il salvataggio in cloud e la sincronizzazione con il cloud stesso ..."
in 
"... che automatizzi il salvataggio in remoto e la sincronizzazione con il cloud ..."